skypy/power_spectrum/tests/test_growth.py

```diff
@@ -57,15 +57,15 @@ def test_growth():
     Dzprime = growth_function_derivative(redshift, cosmology_flat)
 
     # Test growth factor
-    assert redshift.shape == fz.shape,\
+    assert redshift.shape == fz.shape, \
```

Comment:
We would not want to touch the power spectrum module here. Not sure if this has been done by some auto tool?

Reply:
...okay, this appears to be some stricter codestyle which should be fixed by #630